vocabularies/Common.xml

```diff
@@ -1319,6 +1320,21 @@ Upon modification field control logic is invoked so that meta-information like h
         <Annotation Term="Core.LongDescription" String="TODO: add link to UX documentation on https://experience.sap.com/fiori-design/" />
       </Term>
 
+      <Term Name="mediaUploadLink" Type="Edm.String" Nullable="false" AppliesTo="Property">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Common.IsInstanceAnnotation" />
+        <Annotation Term="Core.RequiresType" String="Edm.Stream" />
+        <Annotation Term="Core.IsURL" />
+        <Annotation Term="Core.Description" String="URL for uploading new media content to a Document Management Service" />
+        <Annotation Term="Core.LongDescription">
+          <String>In contrast to the `@odata.mediaEditLink` this URL allows to upload new media content without directly changing a stream property.
+The upload request typically uses HTTP POST with `Content-Type: multipart/form-data` following RFC 7578.
+There should only be one form field for the file. The field name of this form field is arbitrary. An additional `filename` parameter in the form field's `Content-Disposition` header, as described in RFC 7578, is expected.  

```

Comment:
```suggestion
The upload request must contain one multipart representing the content of the file. The `name` parameter in the `Content-Disposition` header (as described in RFC 7578) is irrelevant, but the `filename` parameter is expected.
```
I wouldn't speak of a "form field", because we talk about the request that would result from a form, not about the form itself.